ci/ray_ci/runner.py

```diff
@@ -84,23 +77,24 @@ def _chunk_into_n(list: List[str], n: int):
     return [list[x * size : x * size + size] for x in range(n)]
 
 
+def _get_all_test_query(targets: List[str], team: str) -> str:
```

Comment:
could you add a comment about what this is and/or why this works?

Reply:
Added